lib/experimental/class-wp-rest-navigation-fallbacks-controller.php

```diff
@@ -0,0 +1,161 @@
+<?php
+/**
+ * WP_REST_Navigation_Fallbacks_Controller class
+ *
+ * REST Controller to create/fetch a fallback Navigation Menu.
+ *
+ * @package Gutenberg
+ * @subpackage REST_API
+ * @since 6.3.0
+ */
+
+/**
+ * Import dependencies.
+ */
+require __DIR__ . '/class-wp-navigation-fallbacks-gutenberg.php';
+
+/**
+ * REST Controller to create/fetch a fallback Navigation Menu.
+ */
+class WP_REST_Navigation_Fallbacks_Controller extends WP_REST_Controller {
```

Comment:
```suggestion
class WP_REST_Navigation_Fallback_Controller extends WP_REST_Controller {
```

... since there is always only one fallback.

Reply:
✅